e2e/nomad09upgrade/exec.nomad

```diff
@@ -0,0 +1,20 @@
+job "sleep" {
+  datacenters = ["dc1"]
+
+  group "sleep" {
+    task "sleep" {
+      driver = "exec"
+
+      config {
+        command = "/bin/sleep"
```

Comment:
I think just calling sleep and letting Nomad find the executable is a slightly more portable approach.

```suggestion
        command = "sleep"
```